joomla/joomla-cms
# [6.2] Move 2 worflows input fields translations to global constants to separate from com_content

### Summary of Changes
This pull requests detaches workflow dropdown inputs from com_content component. It is required or other components using Workflows would require using COM_CONTENT translations or its translation prefix.


### Testing Instructions
To test it fully you would require to build a custom component running workflows. Easiest is to check if Workflow input in Article Edit view and Articles list in administration still have the translation like previously:

- Run Translation
- Workflow Stage



### Actual result BEFORE applying this Pull Request
![obraz](https://github.com/joomla/joomla-cms/assets/2000754/e68e24c2-bce4-464e-8de8-10fae55935db)
![obraz](https://github.com/joomla/joomla-cms/assets/2000754/5f105036-33c3-4e0c-b8c5-062eaf8580a8)




### Expected result AFTER applying this Pull Request
The inputs marked in screens above should have the same translation before and after patch.


### Link to documentations
Please select:
